modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/WithIroha.kt

```diff
@@ -13,7 +13,7 @@ import kotlin.reflect.KClass
 @Test
 @Inherited
 annotation class WithIroha(
-    val genesis: KClass<out Genesis> = EmptyGenesis::class,
+    val blocks: Array<KClass<out Genesis>> = [EmptyGenesis::class],
```

Comment:
extend documentation behind that, how it behaves if specify many values, maybe rename to smth more explicit like `blocksToMergeInGenesis` or `transactionSources` or smth